terraform/testnet/addons.tf

```diff
@@ -102,3 +101,44 @@ resource "aws_iam_role_policy" "cluster-autoscaler" {
   role   = aws_iam_role.cluster-autoscaler[0].name
   policy = data.aws_iam_policy_document.cluster-autoscaler[0].json
 }
+
+resource "kubernetes_namespace" "chaos-testing" {
+  count = var.enable_chaos ? 1 : 0
+
+  metadata {
+    annotations = {
+      name = "chaos-testing"
+    }
+
+    name = "chaos-testing"
+  }
+}
+
+resource "helm_release" "chaos-mesh" {
+  count     = var.enable_chaos ? 1 : 0
+  name      = "chaos-mesh"
+  namespace = kubernetes_namespace.chaos-testing[0].metadata[0].name
+
+  chart       = "${path.module}/../helm/chaos"
+  max_history = 10
+  wait        = false
+
+  values = [
+    jsonencode({
+      ingress = {
+        enable                   = true
+        domain                   = "chaos.${local.domain}"
+        acm_certificate          = var.zone_id != "" ? aws_acm_certificate.ingress[0].arn : null
```

Comment:
let's make ACM certificate required.

Reply:
Since the `testnet` TF doesn't require creating a cert, that condition's a bit difficult to enforce. How about we just disable the ingress if there's no cert?